### ISSUE TITLE
R MArkdown LaTex failed to compile new.tex.

### ISSUE THREAD

**Issue by RogerRodrigo1** [yihui/tinytex#182]
I am having serious issues with R Markdown saving my work in PDF form it is giving me the below error can anyone assist?

output file: new.knit.md

"C:/Program Files/RStudio/bin/pandoc/pandoc" +RTS -K512m -RTS new.utf8.md --to latex --from markdown+autolink_bare_uris+tex_math_single_backslash --output new.tex --self-contained --highlight-style tango --latex-engine pdflatex --variable graphics --variable "geometry:margin=1in" 
This is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/W32TeX) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
tlmgr search --file --global "/epstopdf-base.sty"
TeX Live 2019 is frozen forever and will no
longer be updated.  This happens in preparation for a new release.

If you're interested in helping to pretest the new release (when
pretests are available), please read https://tug.org/texlive/pretest.html.
Otherwise, just wait, and the new release will be ready in due time.
Trying to automatically install missing LaTeX packages...
tlmgr install epstopdf-pkg
TeX Live 2019 is frozen forever and will no
longer be updated.  This happens in preparation for a new release.

If you're interested in helping to pretest the new release (when
pretests are available), please read https://tug.org/texlive/pretest.html.
Otherwise, just wait, and the new release will be ready in due time.
Unable to download the checksum of the remote TeX Live database,
but found a local copy, so using that.

You may want to try specifying an explicit or different CTAN mirror,
or maybe you need to specify proxy information if you're behind a firewall;
see the information and examples for the -repository option at
https://tug.org/texlive/doc/install-tl.html
(and in the output of install-tl --help).

tlmgr.pl: package repository http://ctan.mirrors.hoobly.com/systems/texlive/tlnet (not verified: unknown)
[1/1, ??:??/??:??] install: epstopdf-pkg [5k]
TLPDB::_install_package: downloading did not succeed (download_file failed)
tlmgr.pl: package log updated: C:/Users/gan09494/AppData/Roaming/TinyTeX/texmf-var/web2c/tlmgr.log
tlmgr update --self
TeX Live 2019 is frozen forever and will no
longer be updated.  This happens in preparation for a new release.

If you're interested in helping to pretest the new release (when
pretests are available), please read https://tug.org/texlive/pretest.html.
Otherwise, just wait, and the new release will be ready in due time.
Unable to download the checksum of the remote TeX Live database,
but found a local copy, so using that.

You may want to try specifying an explicit or different CTAN mirror,
or maybe you need to specify proxy information if you're behind a firewall;
see the information and examples for the -repository option at
https://tug.org/texlive/doc/install-tl.html
(and in the output of install-tl --help).

tlmgr.pl: package repository http://ctan.mirrors.hoobly.com/systems/texlive/tlnet (not verified: unknown)
tlmgr.pl: no self-updates for tlmgr available.
tlmgr install epstopdf-pkg
TeX Live 2019 is frozen forever and will no
longer be updated.  This happens in preparation for a new release.

If you're interested in helping to pretest the new release (when
pretests are available), please read https://tug.org/texlive/pretest.html.
Otherwise, just wait, and the new release will be ready in due time.
Unable to download the checksum of the remote TeX Live database,
but found a local copy, so using that.

You may want to try specifying an explicit or different CTAN mirror,
or maybe you need to specify proxy information if you're behind a firewall;
see the information and examples for the -repository option at
https://tug.org/texlive/doc/install-tl.html
(and in the output of install-tl --help).

tlmgr.pl: package repository http://ctan.mirrors.hoobly.com/systems/texlive/tlnet (not verified: unknown)
[1/1, ??:??/??:??] install: epstopdf-pkg [5k]
TLPDB::_install_package: downloading did not succeed (download_file failed)
tlmgr.pl: package log updated: C:/Users/gan09494/AppData/Roaming/TinyTeX/texmf-var/web2c/tlmgr.log
This is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/W32TeX) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
tlmgr search --file --global "/epstopdf-base.sty"
TeX Live 2019 is frozen forever and will no
longer be updated.  This happens in preparation for a new release.

If you're interested in helping to pretest the new release (when
pretests are available), please read https://tug.org/texlive/pretest.html.
Otherwise, just wait, and the new release will be ready in due time.
! LaTeX Error: File `epstopdf-base.sty' not found.

! Emergency stop.
<read *> 

Error: LaTeX failed to compile new.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See new.log for more info.
Execution halted

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.name/issue/.
- [x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('tinytex')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('yihui/tinytex')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by yihui** [comment]
Please take a look at your three promises at the end of your post, and fulfill them. Thanks!

**Comment by balachandarkaliappan** [comment]
Did you find the answer @RogerRodrigo1 ?? I have the same issue. I tried updating TinyTex and rmarkdown. But it did not work.

**Comment by yihui** [comment]
@balachandarkaliappan Please see my reply above. Just like @RogerRodrigo1, you didn't provide any information required by the issue guide. Without you providing the necessary information first, there's no way we could help. Thanks!

**Comment by balachandarkaliappan** [comment]
@yihui Thanks for your reply. This is my first question in github - so I did not know how to write it properly. I had similar issue to @RogerRodrigo1. 

While knitting it always throwed a message saying "TeX Live 2019 is frozen forever and will no
longer be updated. This happens in preparation for a new release."

Later, I followed your debugging guidelines given here- https://yihui.org/tinytex/r/#debugging. This solved my issue. Now, I am able to knit documents as pdf file. Thanks!
